features/testbot/bugs.feature

```diff
@@ -0,0 +1,5 @@
+@routing @testbot @bug
+Feature: Known bugs
+
+    Background:
+        Given the profile "testbot"
```

Comment:
is that feature designated to do something? Right now it seems completely empty. If we want to keep it, we should move known bugs here